x-pack/solutions/observability/plugins/slo/public/pages/slo_management/components/slo_management_table.tsx

```diff
@@ -115,6 +108,10 @@ export function SloManagementTable() {
     setSloToDisable(undefined);
   };
 
+  const updateFilter = (newState: SearchState) => {
+    onStateChange(newState);
+  };
+
```

Comment:
if you handle the state from the search bar component directly, no need to have this proxy